content/nginx-one/rbac/roles.md

```diff
@@ -13,13 +13,13 @@ We provide three default **[roles](https://docs.cloud.f5.com/docs-v2/administrat
 
 ### Admin
 
-The Admin role, identified as <code>f5xc-nginx-one-admin</code>, provides full read and write access to all endpoints and features within the NGINX One Console.
+The Admin role, identified as `f5xc-nginx-one-admin`, provides full read and write access to all endpoints and features within the NGINX One Console and related services.
```

Comment:
@tstraley Should we be more specific -- what's a "related service?" (Maybe it's already in the XC docs?)

Reply:
Thanks @mjang -- I'm not sure if that link provides the description of the XC services implied here, but please feel free to make changes as you see fit. I'm just hoping to improve the accuracy of this documentation, so pointing attention to the fact that these default roles aren't purely scoped to NGINX One APIs felt important, but I'm not particular about how that is accomplished here.

Reply:
> so pointing attention to the fact that these default roles aren't purely scoped to NGINX One APIs felt important

The roles in the link are beyond NGINX One APIs, so I think we've met your criteria.